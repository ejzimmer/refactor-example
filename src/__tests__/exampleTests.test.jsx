import moment from "moment"
import { render } from "@testing-library/react"

function formatDate(date) {
  if (!date) return ""

  return moment(new Date(date)).format("yyyy-MM-D")
}

function UserInfo({ user }) {
  const date = formatDate(user.birthday)
  const day = moment(user.birthday).format("dddd")

  return (
    <div>
      {user.name} was born on {date} ({day})
    </div>
  )
}

describe("the test suite", () => {
  it("has at least one test", () => {
    expect(true).toBe(true)
  })
})
