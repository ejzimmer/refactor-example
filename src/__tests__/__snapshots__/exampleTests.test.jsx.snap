// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo component matches the snapshot 1`] = `
<div>
  <div>
    Bob
     was born on 
    2010-02-11
     (
    Thursday
    )
  </div>
</div>
`;
